### ISSUE TITLE
InlineDocCommentDeclaration false positive

### ISSUE THREAD

**Issue by dereuromark** [slevomat/coding-standard#1454]
const on root level (without PHP class file) should be ignore from

> SlevomatCodingStandard.Commenting.InlineDocCommentDeclaration.InvalidFormat

The following code

```php
/**
 * @var string
 */
const APPLICATION_ENV = 'development';
/**
 * @var string
 */
const APPLICATION_STORE = 'DE';
const APPLICATION_ROOT_DIR = __DIR__ . '/cli_sandbox';
const APPLICATION_VENDOR_DIR = __DIR__ . '/../../../../../vendor';
const APPLICATION_SOURCE_DIR = APPLICATION_ROOT_DIR . '/src/';

require_once APPLICATION_VENDOR_DIR . '/autoload.php';

$bootstrap = new Spryker\Zed\Console\Communication\ConsoleBootstrap();
$bootstrap->run();
```
results in
```
FILE: /data/tests/PyzTest/Zed/Console/_data/console_runner.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 4 | ERROR | Invalid inline documentation comment format "@var
   |       | string", expected "@var type $variable Optional
   |       | description".
 8 | ERROR | Invalid inline documentation comment format "@var
   |       | string", expected "@var type $variable Optional
   |       | description".
----------------------------------------------------------------------
```

which it should not, since const are declared as such without the variable.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/4eeb22f8d1171871563f50f8187130f6dc5acce5
